### DISCUSSION TITLE
"No Data Found" on torrents that have been downloaded/seeding. No other changes on system.

### DISCUSSION THREAD

**Discussion by ghost** [transmission/transmission#6951] [Support]
So I've built some new transmission servers and am running into an issue with torrents that have been downloaded and have been seeding fine, then I come back the next day and check and a small percentage of torrents all of a sudden show error with the "No data found" error.

NO other changes have happened on the server. All torrents are seeding, let's say 500 .. then I come back the next morning and 40 or 50 of them are in error. The server has not been rebooted, the daemon service is running fine, the SMB mount with ALL the data is working fine and has not had any hiccups. I don't understand why this is occurring.

A restart of the transmission-daemon service seems to resolve. All 500 torrents will start seeding again. If I initiate a manual 'verify' of these errored torrents then they will go back to seeding. Obviously the data is there and is correct.

Why is transmission doing this? What can I check to see what is causing this? I'm running daemon 4.0.5.

**Comment by killemov** [comment]
> and has not had any hiccups

This assumption.
